pkg/ui/workspaces/cluster-ui/src/badge/badge.module.scss

```diff
@@ -1,4 +1,5 @@
 @import '../core/index.module.scss';
+@import "~@cockroachlabs/design-tokens/dist/web/tokens";
```

Comment:
FYI, `~` in the import is a webpack alias for `node_modules`, which is needed since design-tokens is an installed package
https://stackoverflow.com/questions/43113870/what-is-the-tilde-doing-in-this-javascript-import